From f77fe9ff3a1c03cb1713368a84026175f3ca57c6 Mon Sep 17 00:00:00 2001
From: Ziyi Gai <quic_zgai@quicinc.com>
Date: Wed, 13 Sep 2023 10:54:42 +0800
Subject: [PATCH] gstreamer1.0-plugins-good: modify caps.

Signed-off-by: Ziyi Gai <quic_zgai@quicinc.com>
---
 /gst/avi/gstavimux.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gst/avi/gstavimux.c b/gst/avi/gstavimux.c
index a71c243..c400875 100644
--- a/gst/avi/gstavimux.c
+++ b/gst/avi/gstavimux.c
@@ -99,8 +99,8 @@ static GstStaticPadTemplate video_sink_factory =
         "height = (int) [ 16, 4096 ], "
         "framerate = (fraction) [ 0, MAX ]; "
         "image/jpeg, "
-        "width = (int) [ 16, 4096 ], "
-        "height = (int) [ 16, 4096 ], "
+        "width = (int) [ 16, 8192 ], "
+        "height = (int) [ 16, 8192 ], "
         "framerate = (fraction) [ 0, MAX ]; "
         "video/x-divx, "
         "width = (int) [ 16, 4096 ], "
--
2.25.1

